tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -280,6 +295,8 @@ def _test_cpu_rdmsr(context):
     # Baselines are taken by running `msr_reader.sh` on:
     #  * host running kernel 4.14 and guest 4.14 with the `bionic-msrtools` rootfs
     #  * host running kernel 4.14 and guest 5.10 with the `bionic-msrtools` rootfs
+    #  * host running kernel 5.10 and guest 4.14 with the `bionic-msrtools` rootfs
```

Comment:
I can only see 1 set of baselines added for T2A 4.14 and 5.10 guests. T2S baselines were collected on 4.14 host. Are you saying that T2A baselines were collected on 5.10 host? If that's true, this can be stated explicitly.

Reply:
2 baselines were collected on 4.14 host kernel and 2 were collected on 5.10 host kernel.
But the difference between msr_list_T2A_4.14Host_4.14Guest.csv and msr_list_T2A_4.14Host_5.10Guest.csv showed MSRs that were already part of MSR_EXCEPTION_LIST so I just used 1 of them as msr_list_T2A_4.14.csv. Same for msr_list_T2A_5.10.csv. If there was a difference between baselines created on the same host then maybe we would have had to add them explicitly.

>T2S baselines were collected on 4.14 host.
>
This conflicts with the code we have because as per the below code the 4.14 or 5.10 in the baseline file name refers to the host kernel so ideally we should be collecting baselines on the respective host kernels:

```
 baseline_df = pd.read_csv(
         f"../resources/tests/msr/msr_list_{cpu_template}_{utils.get_kernel_version(level=1)}.csv"
     )
 
```

Reply:
Hm, but reading the commit message https://github.com/firecracker-microvm/firecracker/commit/bbf28d31a364c25f9433572836263504c0b3975f, it now looks to me that the code is correct, but the comment is not...

Reply:
I'm now thinking that encoding both guest and host kernels in the baseline name could be a good idea to avoid further confusion. This would require rebaselining T2S though.

Reply:
yes, it was bit confusing for me at the beginning too. I can collect baselines for T2S as well if you are ok to having baselines for host and guest kernel combination.

Reply:
I have updated the baselines files and made changes to use them. I am keeping original T2S baseline files for future reference.